test/e2e/core.go

```diff
@@ -82,6 +82,7 @@ func CoreDump(dir string) {
 		cmds = append(cmds, []command{
 			{"cat /var/log/kubelet.log", "kubelet"},
 			{"cat /var/log/supervisor/supervisord.log", "supervisord"},
+			{"cat /var/log/dmesg", "dmesg"},
```

Comment:
Instead of collecting /var/log/message, I changed the pr to collect /var/log/kernel.log. The difference between them are:

/var/log/kern.log captures only kernel messages; and /var/log/messages captures more.